packages/@react-spectrum/accordion/test/Accordion.test.js

```diff
@@ -18,17 +18,17 @@ import {theme} from '@react-spectrum/theme-default';
 import userEvent from '@testing-library/user-event';
 
 let items = [
-  {key: 'one', title: 'one title', children: 'one children'},
-  {key: 'two', title: 'two title', children: 'two children'},
-  {key: 'three', title: 'three title', children: <input type="text" />}
+  {id: 'one', title: 'one title', children: 'one children'},
+  {id: 'two', title: 'two title', children: 'two children'},
+  {id: 'three', title: 'three title', children: <input type="text" />}
 ];
 
 function renderComponent(props) {
   return render(
     <Provider theme={theme}>
       <Accordion {...props}>
         {items.map(item => (
-          <Disclosure key={item.key}>
+          <Disclosure key={item.id} id={item.id}>
```

Comment:
i might just be missing some context here but do we still need `key` on Disclosure or is `id` sufficient enough?

Reply:
The key prop is still there (for React map). We use id for our expanded keys, so I kept both.

Reply:
ohh right right